Text/Shakespeare/Base.hs

```diff
@@ -281,6 +284,9 @@ parseUnder = do
         deref <- derefCurlyBrackets
         return $ Right deref) <|> return (Left "_")
 
+readFileUtf8 :: FilePath -> IO String
```

Comment:
I think the naming here is too subtle versus `readUtf8File`. How about `readUtf8FileString` instead?